core/frontend/src/ViewState.ts

```diff
@@ -1300,6 +1346,16 @@ export abstract class ViewState3d extends ViewState {
     return backgroundMapGeometry ? backgroundMapGeometry.cartographicToDb(cartographic, result) : undefined;
   }
 
+  public async rootToCartographicFromGcs(root: XYAndZ, result?: Cartographic): Promise<Cartographic | undefined> {

```

Comment:
These are `@public`. Docs?

Reply:
Added some docs to these; the functions Ray based these on were also not documented so I tried to describe it as best as I understand from the name and the usage.

Reply:
Gonna need a definition of "root space" - is that the same as "spatial" (ask @RBBentley).

Reply:
Ray verified that yes it is spatial space.  I've now documented the functions as such.